tasks/update-ocp-tag/update-ocp-tag.yaml

```diff
@@ -68,7 +65,6 @@ spec:
         # Access the updated image
         updatedFromIndex=$(replace_tag "$(jq -r '.fbc.fromIndex' "$DATA_FILE")")
         updatedTargetIndex=$(replace_tag "$(jq -r '.fbc.targetIndex' "$DATA_FILE")")
-        updatedBinaryImage=$(replace_tag "$(jq -r '.fbc.binaryImage' "$DATA_FILE")")
 
         # if {{OCP_VERSION}} is not set, the original Index will be kept but its ocp version should
         # match base image version.
```

Comment:
two lines below this needs to be removed too

Reply:
done